feed.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="node_modules/bootstrap/dist/css/bootstrap.min.css"
+    />
+    <link rel="stylesheet" href="src/scss/feed.css" />
+    <title>Feed</title>
+  </head>
+  <body>
+    <div class="card" style="width: 50rem">
```

Comment:
This is not the appropriate way to approach navbar in Bootstrap. See the example here of bootstrap navbars at https://getbootstrap.com/docs/5.3/components/navbar/#supported-content